GitUI/CommandsDialogs/BrowseDialog/FormOpenDirectory.cs

```diff
@@ -86,5 +86,26 @@ private void DirectoryKeyPress(object sender, KeyPressEventArgs e)
                 LoadClick(null, null);
             }
         }
+
+        private void folderGoUpbutton_Click(object sender, EventArgs e)
+        {
+            try
+            {
+                var currentFolder = new DirectoryInfo(_NO_TRANSLATE_Directory.Text);
+                if (currentFolder.Exists)
+                {
+                    string parentPath = currentFolder.Parent != null ? currentFolder.Parent.FullName : currentFolder.FullName;
+                    if (parentPath.EndsWith(@"\"))
+                        parentPath = parentPath.TrimEnd('\\');
+                    _NO_TRANSLATE_Directory.Text = parentPath;
+                    _NO_TRANSLATE_Directory.Focus();
+                    _NO_TRANSLATE_Directory.Select(_NO_TRANSLATE_Directory.Text.Length, 0);
+                    SendKeys.Send(@"\");
+                }
+            }
+            catch (Exception)
+            {
+            }
```

Comment:
The `try{} catch` is here because some methods could throw and I don't want to have a crash for a feature that should just be a convenience.